addons/binding/org.openhab.binding.tellstick/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,35 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Tellstick Binding
+Bundle-SymbolicName: org.openhab.binding.tellstick;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jna-4.0.0.jar,
+ lib/javatellstick-1.0.jar,
+ lib/async-http-client-2.0.19.jar,
+ lib/async-http-client-netty-utils-2.0.19.jar,
+ lib/netty-reactive-streams-1.0.8.jar,
+ lib/reactive-streams-1.0.0.jar,
+ lib/netty-all-4.0.42.Final.jar,
+ lib/netty-resolver-2.0.19.jar,
+ lib/netty-resolver-dns-2.0.19.jar
+Import-Package: 
+ com.google.common.collect,
+ org.apache.commons.lang,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.hamcrest;core=split;resolution:=optional,
+ org.junit;version="4.0.0";resolution:=optional,

```

Comment:
I tried to move my tests to a fragment, but I just could not get it to work in Eclipse. I will give it another go.